daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/processors/unparsers/SpecifiedLength2.scala

```diff
@@ -460,6 +470,102 @@ class ElementUnusedUnparser(
 
 }
 
+/**
+ * This is basically a simplified version of the ElementUnusedUnparserSuspendableOperation.
```

Comment:
Do you mean SuppressableSeparatorUnparserSuspendableOperation? This looks much more liket tat that ElementUnused...Operation.

Reply:
Stale comment.  It was originally based on ElementUnused but is now based off Suppressable.